Source/WTF/wtf/text/StringCommon.h

```diff
@@ -38,6 +38,7 @@
 #include <wtf/UnalignedAccess.h>
 #include <wtf/text/ASCIIFastPath.h>
 #include <wtf/text/ASCIILiteral.h>
+#include <wtf/unicode/UTF8Conversion.h>
```

Comment:
This is an unwanted compilation slowdown. I wonder if there's a way we can avoid it.

Reply:
IMHO, they should be in this header file, but I guess I can move the implementation to the .cpp file.